projects/ngrx.io/content/guide/store/architecture.md

```diff
@@ -1 +1,418 @@
 # Architecture
+
+NGRX is based on the architectural pattern Redux. Redux in turn is based upon the general pattern Pub-Sub or Publish, Subscribe. This means we send a message to a central data store and a number of listeners on that data store are being told there is a new message. 
+
+A very typical usage for this is a Chat application. In a Chat application the people in the chat room can communicate with specific participants or they can broadcast what they say to the entire chat room. It's messages being sent, some messages meant for specific people or for a group. 
```

Comment:
- Consider changing `A very typical usage` to `The Redux pattern is a great fit for a chat application.`.
- Missing comma after `In a Chat Application`
- Consider removing `what they say` from `they can broadcast what they say to the entire chat room`.
- `It's messages being sent, some messages meant for specific people or for a group` - Looks like there are some missing words here, should this be `Messages can be sent to either specific people or the entire group`?